### ISSUE TITLE
Change the use of user friendly name to user`s user_id / username

### ISSUE THREAD

**Issue by maksp86** [Nerwyn/material-rounded-theme#55]
**Is your feature request related to a problem? Please describe.**
New material-rounded-theme.js script tries to find user color sensor with invalid entity_id
![image](https://github.com/user-attachments/assets/4438e9e9-c803-4765-82ee-8d1583f81326)
Script using lower`ed friendly name of current user which contains non ascii characters (cyrillic in my case). That is not valid for entity_id in HA.

**Describe the solution you'd like**
Usage of user_id or just username would be more appropriate in that scenario.

**Comment by maksp86** [comment]
Just looked it up, script definetly uses friendly_name from hass.user object
``` TypeScript
const userId = ha.hass.user?.name.toLowerCase().replace(' ', '_');
```
This object also has an id property, so it can be easily used
![image](https://github.com/user-attachments/assets/f2f4bbba-5ad2-4276-afba-e27bae122a63)

**Comment by Nerwyn** [comment]
It looks like there's multiple things I have to fix:
 
1. `hass.connection.subscribeMessage` fails if it's provided a non-existent entity ID, so I have to add some filtering to the array provided to it to check that these entity IDs do exist.
2. An additional sensor option by user ID for users with non-ascii characters in their user friendly name.

We are limited in what we can use to identify the current user by what's available in the `hass` object.

While user name would be ideal, it isn't available in `hass.user`. User ID is though, as you've pointed out.
```json
{
    "id": "abc123uuid",
    "name": "Nerwyn Singh",
    "is_owner": true,
    "is_admin": true,
    "credentials": [
        {
            "auth_provider_type": "homeassistant",
            "auth_provider_id": null
        }
    ],
    "mfa_modules": [
        {
            "id": "totp",
            "name": "Time-based One Time Password",
            "enabled": true
        }
    ]
}
```

I can definitely add this as an option for user specific sensors. The only snag is that I have to figure out an end user friendly way to find this value to put in the README, preferably without using the browser console for less technical people.

**Comment by Nerwyn** [comment]
I've pushed the fix for non-existent sensor entity IDs and using user ID as an alternative to user name in the sensor name to the dev branch. You can either download and use the updated JS module from there or use the [dev branch CDN link](https://cdn.jsdelivr.net/gh/nerwyn/material-rounded-theme@dev/dist/material-rounded-theme.js). I haven't had a chance to test it myself yet but it should work!

**Comment by Nerwyn** [comment]
[Added/fixed in 3.0.1.](https://github.com/Nerwyn/material-rounded-theme/releases/tag/3.0.1)
